README.md

```diff
@@ -2,29 +2,52 @@
 
 ctl10n (compile time localization) provides you a simple way to embed messages
 into binary file without embedding them into source. Internally, ctl10n generates
-simple `macro_rules!` macro `tr!()` from provided TOML file with strings.
+simple `macro_rules!` macro `tr!()` from provided a TOML file with strings.
```

Comment:
Seems like article isn't needed here? "From provided file" is ok

Reply:
Hm, you're right, a bit wrong. Maybe
> Internally, ctl10n generates *the* simple `macro_rules!` macro `tr!()` from *a* provided TOML file with strings.


The original just didn't sound right to me.

Reply:
It probably should be 

> generates _a_ simple `macro_rules!` macro `tr!()` from _the_ provided TOML file with strings

I will fix this and merge PR now.